v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/TextIOToBigQuery.java

```diff
@@ -215,7 +215,7 @@ public interface Options
         optional = false,
         regexes = {"^gs:\\/\\/[^\\n\\r]+$"},
         description = "The GCS location of the text you'd like to process",
-        helpText = "The path to the Cloud Storage text to read.",
+        helpText = "Cloud Storage location of the text you'd like to process.",
         example = "gs://your-bucket/your-file.txt")
```

Comment:
Question: Will the `example ` field be pulled into the parameter descriptions in the doc? (Otherwise the docs will lose the examples)

Not a blocker for this PR

Reply:
Yes, example will be pulled to readme file.